app/models.py

```diff
@@ -86,6 +87,7 @@ def serialize(self):
             'slug': self.slug,
             'framework': self.framework,
             'status': self.status,
+            'clarification_questions_open': self.clarification_questions_open,
```

Comment:
Other serializations use camelCase for keys - e.g. `frameworkSlug`, `contactName`, `phoneNumber`, `dunsNumber`, etc.